pointcloud: True
intensity_image: True

exposure_mode: DEFAULT #, AUTO, AUTO_HDR, MANUAL, MANUAL_HDR
exposure: 5000